Library/Dependencies/napi/source/js_native_api_JavaScriptCore.cc

```diff
@@ -1,97 +1,220 @@
 #include <napi/js_native_api.h>
+#include <napi/js_native_api_types.h>
+#include "JavaScriptCore/JavaScriptCore.h"
+#include "js_native_api_JavaScriptCore.h"
+#include <string>
+#include <vector>
+#include <map>
+#include <napi/napi.h>
+
+struct VFTableEntry
```

Comment:
let's get on a call and go over this. mucking with vtables is difficult to review without context ;)

Reply:
I thought this was a vtable, but it's actually not, but nevertheless we talked and I will continue the review now.